codeql/windows-drivers/queries/Windows/wdk/kmdf/DeviceInitApi/test/pass/library/fail_library1.c

```diff
@@ -0,0 +1,98 @@
+/*++
```

Comment:
Are the fail drivers in the PASS library the same as the ones in the FAIL library?
Whats the difference between the Pass/Fail directories if so?

Reply:
The fail folder has an injected defect for this query, while the pass folder does not.  I did not rename the individual files in the pass folder in part because these full drivers are stopgaps until we implement Abenezer+Jakob's code snippet+template test case style.

Reply:
We will need to discuss how to separate test code from queries code in the hierarchy to make it easier to check in for release. We dont want customers accidentally downloading a bunch of test code. But I see your point and agree.